dev/RadioButtons/RadioButtonsPrimitives.idl

```diff
@@ -1,19 +1,33 @@
-namespace MU_XCP_NAMESPACE
+﻿namespace MU_XCP_NAMESPACE
 {
-
 [WUXC_VERSION_PREVIEW]
 [webhosthidden]
-unsealed runtimeclass RadioButtonsListView : Windows.UI.Xaml.Controls.ListView
+[default_interface]
+unsealed runtimeclass RadioButtonsElementFactory : MU_XC_NAMESPACE.ElementFactory
```

Comment:
Are these types expected to be public ?

Reply:
Currently they exist in the control template. If we want to keep that then they will need to be public.  We could set these from code behind if we did not want to make them public.

---
In reply to: [347418539](https://github.com/microsoft/microsoft-ui-xaml/pull/1623#discussion_r347418539) [](ancestors = 347418539)